environment

```diff
@@ -2,6 +2,7 @@
 set -a
 
 PROJECT_ROOT=`pwd`
+DEFAULT_SLACK_CHANNEL='dcp-ops-alerts'
```

Comment:
Please put this in Secrets Manager

Reply:
oh sorry misunderstood your other comment